### ISSUE TITLE
Audit #1 – General Overview when logged in – Main layout column on desktop has no explicit background color

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4331]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
Users sometimes use custom stylesheets to improve their perception of web interfaces, for example change the general background color of `html`. But when you apply a style like `html {background-color:black;color:white}` on the interface, the text in the main column (on desktop) becomes unreadable (dark blue on black).

# Remediation
The main column (on desktop) needs an explicit color styling in CSS (white/`#fff`) in this case so it isn't `transparent` anymore.

**Comment by LukasHirt** [comment]
@marcus-herrmann Not sure if we understand this issue. Does this mean that we need to prevent users from changing the colour of the background for the content?

**Comment by marcus-herrmann** [comment]
@LukasHirt No, in the contrary. Set the background of `.uk-width-expand` (the main column on desktop) to white, instead of the implicit `transparent`

**Comment by pascalwengerter** [comment]
Closed via #4862
